src/core/utilities.js

```diff
@@ -59,7 +57,7 @@ function extend( a, b, undefOnly ) {
 		if ( hasOwn.call( b, prop ) ) {
 
 			// Avoid "Member not found" error in IE8 caused by messing with window.constructor
-			if ( !( prop === "constructor" && a === window ) ) {
+			if ( !( prop === "constructor" && a === global ) ) {
```

Comment:
`window` is correct here, but this leads to a different check. This block runs on every env and global is set to avoid errors on node.